### ISSUE TITLE
Problem dep installing on a compute node

### ISSUE THREAD

**Issue by sierra-tango-echo** [alces-software/clusterware#130]
via a qrsh session, load the pyprophet module

```
[alces@ip-10-75-0-126(steviewonder) ~]$ module load apps/pyprophet/0.18.3/gcc-4.8.5+python-2.7.8+numpy-1.9.2+pandas-017.0+numexpr-2.4.4+scipy-0.17.0+matplotlib-1.4.3+setuptools-2.1
apps/pyprophet/0.18.3/gcc-4.8.5+python-2.7.8+numpy-1.9.2+pandas-0.17.0+numexpr-2.4.4+scipy-0.17.0+matplotlib-1.4.3+setuptools-2.1
 | -- libs/gcc/system
 |    * --> OK
 | -- apps/python/2.7.8/gcc-4.8.5Installing distro dependency (1/11): openssl-devel ... FAILED
FAILED: unable to load apps/python/2.7.8/gcc-4.8.5
Installation of required distribution packages failed, please refer to: /opt/gridware/local/el7/etc/depends/apps-python-2.7.8-gcc-4.8.5.sh
 |    * --> FAILED apps/python/2.7.8/gcc-4.8.5 --

```

**Comment by sierra-tango-echo** [comment]
hmm, i took --quiet out of the yum line on the script and then the script ran

**Comment by mjtko** [comment]
Seems to be related to the default hard memory limit (`h_vmem`) of 1G not being enough for `yum` to complete its cache set up operation successfully.

**Comment by mjtko** [comment]
Should be fixed via alces-software/clusterware-services#23.
